tests/GatewayTest.php

```diff
@@ -178,6 +178,40 @@ public function test_failing_tokenization() {
 		$this->assertEquals($result->errors[0], "Card number is invalid");
 	}
 
+	/**
+	 * Testing fetching a previously tokenized card
+	 */
+	public function test_get_tokenized_card() {
+		$gw = new FatZebra\Gateway("TEST", "TEST", true);
+		$gw->set_timeout(30);
+
+		$result = $gw->tokenize("Joe Bloggs", "5123456789012346", "05/2023", "123");
+
+		$this->assertTrue($result->successful);
+		$this->assertNotNull($result->response->token);
+
+		$token = $result->response->token;
+		$result = $gw->get_tokenized_card($token);
+
+		$this->assertTrue($result->successful);
+		$this->assertEquals($result->response->card_holder, "Joe Bloggs");
```

Comment:
I think when your inputs are also your expected  outputs like in this test then they should become a variable defined at the start of the test you reference to avoid any differences in either side such as a typo.

Reply:
done.